merlin/process.py

```diff
@@ -143,13 +144,13 @@ def __init__(self, process, view):
         self.process = process
         self.view = view
 
-    def send_query(self, *query):
+    def send_query(self, *query, verbosity=None):
```

Comment:
If verboisty isn't used in other commands, any reason it couldn't just be part of the query?

Reply:
So verbosity is used in the other commands, I just haven't figured out how. Also verbosity isn't part of the query object, its external to that